src/extensions/default/CodeFolding/main.js

```diff
@@ -263,34 +267,38 @@ define(function (require, exports, module) {
       */
     function createGutter(editor) {
         var cm = editor._codeMirror;
+        var rootElement = editor.getRootElement();
         var path = editor.document.file.fullPath, _lineFolds = prefs.getFolds(path);
         _lineFolds = _lineFolds || {};
         cm._lineFolds = _lineFolds;
         var gutters = cm.getOption("gutters").slice(0);
 
         // Reuse any existing fold gutter
         if (gutters.indexOf(GUTTER_NAME) < 0) {
-            var lnIndex = gutters.indexOf("CodeMirror-linenumbers");
-            $(editor.getRootElement()).addClass("folding-enabled");
-            gutters.splice(lnIndex + 1, 0, GUTTER_NAME);
+            var lineNumberIndex = gutters.indexOf(LINE_NUMBER_GUTTER);
+            if (lineNumberIndex < 0) {
+                $(rootElement).addClass("linenumber-disabled");
```

Comment:
@MarcelGerber originally wrote : 

>This might also go well here: https://github.com/adobe/brackets/blob/7814da193b98ff7889714a601d630973bddf02d0/src/editor/Editor.js#L2324-L2328

>That way, stylesheets and other extension have an easy way to reuse this state.
cc @nethip